src/Dial.svelte

```diff
@@ -1,105 +1,77 @@
-<script>
-import { onMount } from 'svelte';
-import Button from './Button.svelte';
-
-let leftsize = 300
-let gutter = 10
-
-let resizing = -1
-
-
-function resizeStart(e) {
-  resizing = e.pageX;
-}
-
-function resizeStop(e) {
-  resizing = -1
-}
-
-function resize(e) {
-  if (resizing >= 0) {
-    var i = e.pageX - resizing
-    leftsize += i 
-    console.log(i)
-    resizing = e.pageX;
-  }
-}
-
-</script>
-<div id="dial">
-  <nav>
-    <Button icon="pacman" />
-    <input type="number" bind:value={leftsize}/>
-  </nav>
-  <div id="leftside" style="width: {leftsize}px">
-    <textarea id="source"
-       on:mousedown={resizeStart}
-       on:mouseup={resizeStop}
-      ></textarea>
+<div
+  class="wrapper"
+>
+  <div
+    bind:this={left}
+    class="left">
+    <textarea></textarea>
   </div>
-  <div id="handle" style="left: {leftsize}px" 
-       on:mousedown={resizeStart}
-       on:mouseup={resizeStop}
-       on:mousemove={resize}>
-  </div>
-  <div id="rightside" style="left: {leftsize + gutter}px">
-    <textarea id="dia" disabled
-       on:mousedown={resizeStart}
-       on:mouseup={resizeStop}
-      ></textarea>
+  <div
+    class="separator"
+    on:mousedown={onMouseDown}
+  />
+  <div 
+    bind:this={right}
+    class="right">
   </div>
 </div>
-<style type="text/sass">
 
+<script>
+  import { onMount } from "svelte"
+  let lastSeparatorLocation, left, right
+  onMount(() => {
+    window.addEventListener("resize", onMouseUp)
+  })
+  const onMouseDown = (e) => {
+    lastSeparatorLocation = {
+      e,
+      leftWidth: left.offsetWidth,
+      rightWidth: right.offsetWidth,
+    }
+    window.addEventListener("mousemove", onMouseMove)
```

Comment:
Use `svelte` bind instead of `window.addEventListener`. please respect what you learned recently.